ansible_collections/arista/avd/roles/eos_validate_state/tasks/bgp_check.yml

```diff
@@ -37,7 +37,7 @@
   assert:
     that:
       - bgp_summary.stdout[0].vrfs.default.peers[bgp_neighbor.key].peerState == 'Established'
-    fail_msg: Session state "{{ bgp_summary.stdout[0].vrfs.default.peers[bgp_neighbor.key].peerState }}"
+    fail_msg: Session state {{ bgp_summary.stdout[0].vrfs.default.peers[bgp_neighbor.key].peerState }}
```

Comment:
I think we should wrap the entire "fail_msg" value in double quotes. So we ensure that it is a single string.